lib/engine/game/g_1880/stock_market.rb

```diff
@@ -31,6 +31,14 @@ def move_left(corporation)
 
           super
         end
+
+        def sort_market_legend?
+          @sort_market_legend ||= true
```

Comment:
what's the point of @sort_market_legend ||= true ? same with the false version

Reply:
not needed anymore